public/assets/cpu.js

```diff
@@ -82,14 +82,39 @@ function CPU(memamt) {
     this.keyLoad = 'load';
     this.keyMem = 'memoryOrder';
 
-    this.instruction_amounts = {
+    // stores the current num of instructions in one clock cycle
+    this.curr_instructions = {
         'arithmetic' : 0,
         'controlTransfer' : 0,
         'store' : 0,
         'load' : 0,
         'memoryOrder' : 0
     }
 
+    // stores the exponential rolling average of each type of instr
+    this.instruction_amounts = {
+        'arithmetic' : {
```

Comment:
Don’t worry about it but remember you don’t need quote strings in JavaScript objects 

Otherwise lgtm